.github/workflows/cd.yml

```diff
@@ -36,7 +36,16 @@ jobs:
         run: yarn create-release-notes
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - name: 'Get message'
+        id: 'deploy_message'
+        run: |
+          echo "::set-output name=commit_msg::$(git log -1 --format=%B)"
+          echo "::set-output name=commit_sha::$(git log -1 --format=%H)"
```

Comment:
will this say in the message which package got deployed?

Reply:
should be like this

```
wes$ git log -1 --format=%B
Publish

 - @mux/mux-elements-codemod@1.1.1
 - @mux/mux-player-react@0.1.0-beta.23
 - @mux/mux-player@0.1.0-beta.23
 - @mux/mux-uploader-react@0.1.0-beta.2
 - @mux/mux-uploader@0.1.0-beta.3
 ```